### PR TITLE
Update test_examples

### PR DESCRIPTION
**Pull request by andrey-churkin**
### Changes

Add quantization_aware_training_tensorflow_mobilenet_v2 to test scope

### CODE REVIEW

**Comment by andrey-churkin** [comment]
Build # 645 (NNCF/nightly/test_examples)
tests.cross_fw.examples.test_examples.test_examples[quantization_aware_training_tensorflow_mobilenet_v2] --  25 min

**Comment by andrey-churkin** [comment]
> Please update test duration
> 
> https://github.com/openvinotoolkit/nncf/blob/438871e5bc16bfd2e31d3baf49c9e2c1fffe7af2/tests/cross_fw/examples/.test_durations#L18
> 
> and provide a link with passed test_examples

@alexsu52 Done, Please review

**Comment by andrey-churkin** [comment]
Build # 646 (with batch_size = 32)

**Comment by andrey-churkin** [comment]
| batch_size        | learning_rate | optimizer | metric (int8_top1) |
| :---------------- | :------------ | :-------- |:-------------------|
| 16                | 1e-5          | RMSProp   | 0.9673885107040405 |
| 32                | 1e-5          | RMSprop   | 0.971464991569519  |
| 64                | 1e-5          | RMSprop   | 0.971464991569519  |
| 128               | 1e-5          | RMSprop   | 0.9712101817131042 |
| 16                | 1e-4          | RMSprop   | 0.9449681639671326 |
| 32                | 1e-4          | RMSprop   | 0.9518471360206604 |
| 64                | 1e-4          | RMSprop   | **0.9727388620376587** |
| 128               | 1e-4          | RMSprop   | 0.9678980708122253 |


| batch_size        | learning_rate | optimizer | metric (int8_top1) |
| :---------------- | :------------ | :-------- |:-------------------|
| 16                | 1e-5          | Adam      | 0.9691720008850098 |
| 32                | 1e-5          | Adam      | 0.971464991569519  |
| 64                | 1e-5          | Adam      | 0.9712101817131042 |
| 128               | 1e-5          | Adam      | **0.9727388620376587** |
| 16                | 1e-4          | Adam      | 0.9296815395355225 |
| 32                | 1e-4          | Adam      | 0.95286625623703   |
| 64                | 1e-4          | Adam      | 0.9656050801277161 |
| 128               | 1e-4          | Adam      | 0.9643312096595764 |

File: examples/quantization_aware_training/tensorflow/mobilenet_v2/main.py [hunk 1]
```diff
@@ -150,12 +150,12 @@ def transform_fn(data_item):
...
 
 tf_quantized_model.compile(
-    optimizer=tf.keras.optimizers.Adam(learning_rate=1e-5),
+    optimizer=tf.keras.optimizers.RMSprop(learning_rate=1e-4),
     loss=tf.keras.losses.CategoricalCrossentropy(),
     metrics=[tf.keras.metrics.CategoricalAccuracy()],
 )
 
-tf_quantized_model.fit(train_dataset, epochs=3, verbose=1)
+tf_quantized_model.fit(train_dataset, epochs=1, verbose=1)
```
Comment (alexsu52) [inline examples/quantization_aware_training/tensorflow/mobilenet_v2/main.py:158]:
```suggestion
# To minimize the example's runtime, we train for only 1 epoch. This is sufficient to demonstrate
# that the quantized model produced by QAT is more accurate than the one produced by PTQ.
# However, training for more than 1 epoch would further improve the quantized model's accuracy.
tf_quantized_model.fit(train_dataset, epochs=1, verbose=1)
```

**Review by alexsu52 — CHANGES_REQUESTED** [review]
The test_examples build failed.
https://github.com/openvinotoolkit/nncf/actions/runs/12902524043

**Comment by andrey-churkin** [comment]
New results:

**CPU:**

| batch_size        | learning_rate | optimizer | metric (int8_top1) |
| :---------------- | :------------ | :-------- |:-------------------|
| 16                | 1e-5          | RMSProp   | 0.9678980708122253 |
| 32                | 1e-5          | RMSprop   | 0.9719745516777039 |
| 64                | 1e-5          | RMSprop   | **0.9737579822540283** |
| 128               | 1e-5          | RMSprop   | 0.9704458713531494 |
| 16                | 1e-4          | RMSprop   | 0.9492993354797363 |
| 32                | 1e-4          | RMSprop   | 0.9582165479660034 |
| 64                | 1e-4          | RMSprop   | 0.9607643485069275 |
| 128               | 1e-4          | RMSprop   | 0.9699363112449646 |

**1 GPU**

| batch_size        | learning_rate | optimizer | metric (int8_top1) |
| :---------------- | :------------ | :-------- |:-------------------|
| 16                | 1e-5          | RMSProp   | 0.968917191028595  |
| 32                | 1e-5          | RMSprop   | 0.9729936122894287 |
| 64                | 1e-5          | RMSprop   | 0.9717197418212891 |
| 128               | 1e-5          | RMSprop   | 0.9709554314613342 |
| 16                | 1e-4          | RMSprop   | 0.9475159049034119 |
| 32                | 1e-4          | RMSprop   | 0.9615286588668823 |
| 64                | 1e-4          | RMSprop   | 0.9666242003440857 |
| 128               | 1e-4          | RMSprop   | 0.9681528806686401 |

File: examples/quantization_aware_training/tensorflow/mobilenet_v2/main.py (hunk 1 above)
Comment (andrey-churkin) [reply examples/quantization_aware_training/tensorflow/mobilenet_v2/main.py:161]: NNCF/nightly/test_examples: Build # 648

Comment (andrey-churkin) [reply examples/quantization_aware_training/tensorflow/mobilenet_v2/main.py:161]: PTQ drop: ~0.018
QAT drop: ~0.014

**Comment by andrey-churkin** [comment]
test_examples (GA): https://github.com/openvinotoolkit/nncf/actions/runs/12927090409

**Review by alexsu52 — APPROVED** [review]
